recipes/pytng/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pytng" %}
+{% set version = "0.2.2a1" %}
```

Comment:
This is an alpha release and should probably follow https://github.com/conda-forge/cfep/blob/master/cfep-05.md

However, we cannot do that in staged-recipes. Also. this package does not have a stable release yet. So let's merge it and ask the maintainers to implement CFEP-05 in the feedstocks.